ganga/GangaLHCb/Lib/RTHandlers/GaudiExecRTHandlers.py

```diff
@@ -165,13 +165,7 @@ def prepareCommand(app):
             raise ApplicationConfigurationError("The filetype: %s is not yet supported for use as an opts file.\nPlease contact the Ganga devs is you wish this implemented." %
                                                 getName(opts_file))
 
-    #Check if this was checked out with LbEnv or not
-    isLbEnv = False
-    with open(app.directory+'/Makefile', "r") as makefile:
-        if 'LbEnv' in makefile.read():
-            isLbEnv = True
-
-    sourceEnv = app.getWNEnvScript(isLbEnv)
+    sourceEnv = app.getWNEnvScript(True)
```

Comment:
This fixes #1897 . LbEnv has been mandatory for so long now that this shouldn't be an issue.